charts/nutanix-csi-storage/values.yaml

```diff
@@ -19,28 +19,34 @@ openshift:
   workerIscsiConfig: false
 
 # kubeletDir allows overriding the host location of kubelet's internal state.
+# for NKE cluster set it to "/var/nutanix/var/lib/kubelet"
 kubeletDir: "/var/lib/kubelet"
 
 # Global Settings for all pods
 
+# csiSecretDir is the directory where PC (Prism Central) secret is mounted.
+csiSecretDir: "/var/run/ntnx-secret-dir"
+
+# category configmap name and namespace
+csiCategoryConfigmapName: ntnx-cluster-configmap
+csiCategoryConfigmapNamespace: ntnx-system
+
 nodeSelector: {}
 tolerations: []
 imagePullPolicy: IfNotPresent
 
 # Storage Class settings
 #
 # choose for which mode (Volume, File, Dynamic File) storageclass need to be created
-volumeClass: false
+volumeClass: true
```

Comment:
@subodh01 
volumeClass is now set to true is it expected

Reply:
We should revert back to false. It is better if user creates storageclass outside of helm.

Reply:
done